frontend/test/pytest/test_autograph.py

```diff
@@ -1718,5 +1723,30 @@ def f(param):
         assert f(0.1) == 3
 
 
+@pytest.mark.tf
```

Comment:
This is not correct anymore, since we don't use tensorflow for autograph anymore. @rauletorresc could you remove this marker from all tests in a follow up PR? :)

Reply:
Done (https://github.com/PennyLaneAI/catalyst/pull/709)